src/api/chainHead_unstable_follow.md

```diff
@@ -129,6 +129,148 @@ All items in `finalizedBlockHashes` and `prunedBlockHashes` are guaranteed to ha
 
 The current best block, in other words the last block reported through a `bestBlockChanged` event, is guaranteed to either be the last item in `finalizedBlockHashes`, or to not be present in either `finalizedBlockHashes` or `prunedBlockHashes`.
 
+### operation-body-done
+
+```json
+{
+    "event": "done",
```

Comment:
nit: Should the event here be `operation-body-done` to keep in sync with the explination from below and with `"event": "operation-storage-items"`?